docs/source/hardware/hardware_overview.rst

```diff
@@ -3,129 +3,73 @@
-    - haswell
-    - 4
-  * - E5-2680v3\@\2.80GHz
-    - ivybridge
-    - 1
-  * - E5-2680v4\@\2.40GHz
-    - broadwell
-    - 526
-  * - E5-2690v3\@\2.60GHz
-    - haswell
-    - 200
-  * - E5-4640v2\@\2.20GHz
-    - ivybridge
-    - 1
-  * - E7-4830v3\@\2.10GHz
-    - haswell
-    - 1
-  * - E7-8867v3\@\2.50GHz
-    - haswell
-    - 1
-  * - Gold6132\@\2.60GHz
-    - skylake_avx512
-    - 23
-  * - Gold6140\@\2.60GHz
-    - cascadelake
-    - 1
-  * - Gold6140CPU\@\2.30GHz
-    - skylake_avx512
-    - 2
-  * - Platinum8160\@\2.10GHz
-    - skylake_avx512
-    - 2
-  * - Platinum8276CPU\@\2.20GHz
-    - cascadelake
-    - 156
-  * - XeonGold6132\@\2.60GHz
-    - skylake_avx512
-    - 3
+  * - Feature Name
+    - Number of Nodes 
+    - RAM memory 
+  * -
+    - public, private, total
+    - per node
+  * - skylake
+    - 0, 170, 170
+    - 186 - 3094 GB
+  * - zen2
+    - 40, 292, 332
+    - 256 - 2000 GB
+  * - zen
+    - 40, 300, 340
+    - 256 - 2000 GB
+  * - ivybridge
+    - 64, 130, 194
+    - 31 - 1031 GB
+  * - sandybridge
+    - 8, 0, 8
+    - 384 GB
+  * - haswell
+    - 230, 62, 292
+    - 109 - 1031 GB
+  * - broadwell
+    - 756, 226, 982
+    - 128 - 515 GB
+  * - cascadelake
+    - 260, 88, 340
+    - 186 - 3094 GB
 
 If you are looking for information about GPUs, see :ref:`working_gpus`.
 
-If you are looking for information about the partitions on Discovery, see :ref:`partition_names`.
+If you are interested in more information about the different partitions on NEU's HPC, including the number of nodes per partition, running time limits, job submission limits, and RAM limits, see :ref:`partition_names`. 
 
 
 Using the ``--constraint`` flag
 ================================
-When using ``srun`` or ``sbatch``, you can specify specific hardware features as part of your job by using the ``--constraint=`` flag. Currently,
-there are two supported options with the ``--constraint=`` flag: ``ib`` and ``micro-architecture``.
-For example, if you want to only include nodes that are connected by InfiniBand (IB) with a job that needs to use multiple nodes, you can
-specify ``--constraint=ib`` in your ``srun`` command or as a line in your ``sbatch`` script. Using a constraint can mean that you
-will wait longer for your job to start, as the scheduler (Slurm) will need to find and allocate the appropriate hardware that you have
-specified for your job. For more information about running jobs, see :ref:`using_slurm`.
+When using ``srun`` or ``sbatch``, you can specify hardware features as part of your job by using the ``--constraint=`` flag. This may be particularily useful when benchmarking or if you're using code that was compilied on a certain micro-architechture. Currently, you can use the ``--constraint=`` flag to restrict your job to a specific feature name (e.g., ``haswell``, ``ivybridge``) or you can use the flag: ``ib`` to only include nodes that are connected by InfiniBand (IB) with a job that needs to use multiple nodes. 
```

Comment:
compilied -> compiled

Reply:
when benchmarking or -> when benchmarking, optimizing, or....

Reply:
So are we suggesting that they should use "--constraint=ib" only when dealing with MPI type jobs, not otherwise?

Reply:
I don't know. Would there be a reason to try using "ib" but request a single node?

Reply:
@SerenaCaplins, the "--constraint=ib" ensured that users got one of the d-nodes, which are newer and faster compared to the c-nodes. So that was the main reason for some users to use that flag even though their work required only a single node. However, now that we are advertising other micro-architectures to be used instead, so I think it would be fine to advertise this as is.
No changes required.